.vscode/settings.json

```diff
@@ -5,9 +5,7 @@
     "**/node_modules": true,
     "**/build": true
   },
-  "editor.formatOnSave": true,
```

Comment:
sure? 😀

Reply:
Format on save may conflict with eslint fix, if editor settings for prettier differ

Reply:
seems silly to me to have editor specific config files checked in, but Iæm not losing sleep over it 🤷‍♂